app/models/chargeback.rb

```diff
@@ -8,7 +8,7 @@ class Chargeback < ActsAsArModel
 
   def self.build_results_for_report_chargeback(options)
     _log.info("Calculating chargeback costs...")
-    @options = options
+    @options = options = ReportOptions.new_from_h(options)
```

Comment:
@isimluk why are not you using only `@options` ? even `options` changed his type maybe it can be confusing